_sass/components/_main-grid.scss

```diff
@@ -0,0 +1,31 @@
+.c-main-grid {
```

Comment:
do you want this file with this PR?

Reply:
hmm no? I think I accidentally might have pushed a change in it in this footer branch